app/javascript/App.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import './index.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from './redux/slices/greetingsSlice';
+
+const App = () => {
+  const dispatch = useDispatch();
+  const { list, errors} = useSelector((state) => state.greetings)
+
+  useEffect(() => {
+    dispatch(fetchGreetings());
+  }, [dispatch]);
+
+  function clickHandler() {
+    dispatch(fetchGreetings());
+  }
+
+  function randomGreeting() {
+    let randomIndex = Math.floor(Math.random() * list.length);
+    if (list[randomIndex]) {
+      return list[randomIndex].text;
+    }
+  }
+  return (
+    <div id="greeting-ctr">
+      <h2 id="greeting">{randomGreeting()}</h2>
```

Comment:
- [ ] 👍 Great progress on the frontend! To align with project requirements, consider adding a `Greeting` component and configuring the `App` component with a route that leads to the `Greeting` component. Additionally, integrate `React Router` for smoother navigation. 


![image](https://github.com/Movgto/rails-react-greetings/assets/55840999/686ad2df-1a96-42f0-a8fe-2d820229f6cb)

This adjustment ensures that your `App` component has a route leading to the `Greeting` component. Remember to update your frontend API endpoint as needed. Keep up the good work! 🌟🚀